scaffoldmaker/meshtypes/meshtype_3d_sphereshell1.py

```diff
@@ -98,6 +102,7 @@ def generateBaseMesh(region, options):
         elementsCountUp = options['Number of elements up']
         elementsCountThroughWall = options['Number of elements through wall']
         useCrossDerivatives = options['Use cross derivatives']
+        useCubicHermite = not(options['Use linear through wall'])
```

Comment:
Rename this `useCubicThroughWall`, since cubic still applies elsewhere even if linear through wall.